deploy/kubedirector/kubedirector.hpe.com_kubedirectorclusters_crd.yaml

```diff
@@ -495,6 +495,21 @@ spec:
                             type: array
                         type: object
                     type: object
+                  volumeProjections:
+                    type: array
+                    items:
+                      type: object
+                      required: [volumeName, mountPath]
+                      properties:
+                        volumeName:
```

Comment:
Sorry, one more late comment! I'm wondering if volumeName might be confusing since volumeName is also a property in the PVC that refers to the PV. Maybe something like pvcName would be better?